_layouts/base.html

```diff
@@ -3,17 +3,16 @@
 <!--
     ============================================================================
 
-    Hey! If you're viewing this, you should probably come work on our Technology
-    & Innovation team. We're always looking for a few great designers,
-    developers, data scientists, and network, infrastructure, privacy and
-    security pros. Keep an eye on our job opportunities at:
-    http://www.consumerfinance.gov/jobs/
+    Hey! If you're viewing this, you should probably come work on our
+    Technology & Innovation team. We're always looking for a few great
+    designers; developers; data scientists; and network, infrastructure,
```

Comment:
Why not just, "We're always looking for a few great designers, developers, data scientists, network specialists, infrastructure engineers, privacy geeks, and security pros to join our team."?

Also, is it silly to wonder if anyone would find the term "geeks" pejorative?